src/app/components/pages/settings/nodes/change-url/change-node-url.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Inject, OnInit, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+
+import { CoinService } from '../../../../../services/coin.service';
+
+@Component({
+  selector: 'app-change-node-url',
+  templateUrl: './change-node-url.component.html',
+  styleUrls: ['./change-node-url.component.scss'],
+})
+export class ChangeNodeURLComponent implements OnInit {
+  form: FormGroup;
+
+  constructor(
+    @Inject(MAT_DIALOG_DATA) private data: {coinId: number, url: string},
+    public dialogRef: MatDialogRef<ChangeNodeURLComponent>,
+    private formBuilder: FormBuilder,
+    private coinService: CoinService
+  ) {}
+
+  ngOnInit() {
+    this.initForm();
+  }
+
+  closePopup() {
+    this.dialogRef.close();
+  }
+
+  change() {
```

Comment:
this function (or coinservice) should probably update `nodeURLs` global var to allow requests to new url (I tried to update it to local node but it didn't work, also when changing nodeURLs manually, not sure what exactly was wrong)

Reply:
Fixed. It probably won’t work when using a local node because the node rejects connections from different sources.